components/alert.css

```diff
@@ -0,0 +1,60 @@
+:root {
+  --success-alert-background: #e8fcf1;
+  --info-alert-background: #e6f7ff;
+  --warning-alert-background: #fefbe6;
+  --error-alert-background: #fef2f1;
```

Comment:
can use a bit darker shade for background colors

Reply:
Done.